### ISSUE TITLE
[graphql] deploy fails when using a reference type in the annotations of portable text

### ISSUE THREAD

**Issue by PSalaun** [sanity-io/sanity#1521]
**Describe the bug**

When deploying the GraphQL API from the nextjs landing page template, I get the following error:
![image](https://user-images.githubusercontent.com/36310175/65944112-4c87a800-e431-11e9-86a1-b223a851c303.png)

It comes from the file `blocks/internalLinks.js`:
```
import React from 'react';

const InternalLinkRender = ({ children }) => <span>{children} 🔗</span>;

export default {
  title: 'Internal link to another document',
  name: 'internalLink',
  type: 'reference',
  description: 'Locate a document you want to link to',
  to: [{ type: 'page' }, { type: 'route' }],
  blockEditor: {
    icon: () => '🔗',
    render: InternalLinkRender,
  },
};
```

which is used in `portableText.js`:
```
export default {
  title: 'Portable Text',
  name: 'portableText',
  type: 'array',
  of: [
    {
      type: 'block',
      styles: [
        { title: 'Normal', value: 'normal' },
        { title: 'H1', value: 'h1' },
        { title: 'H2', value: 'h2' },
        { title: 'H3', value: 'h3' },
        { title: 'H4', value: 'h4' },
        { title: 'Quote', value: 'blockquote' },
      ],
      lists: [{ title: 'Bullet', value: 'bullet' }],
      marks: {
        decorators: [
          { title: 'Strong', value: 'strong' },
          { title: 'Emphasis', value: 'em' },
          { title: 'Code', value: 'code' },
        ],
        annotations: [
          { type: 'link' },
          { type: 'internalLink'},
        ],
      },
    },
    { type: 'figure' },
    { type: 'embedHTML' },
  ],
};
```

The only way I've found to get around this issue is to import directly `internalLink` in `portableText`, and remove it from the schema.js OR declare it inline in `portableText`. Example:
```
import internalLink from './internalLink';

export default {
  title: 'Portable Text',
  name: 'portableText',
  type: 'array',
  of: [
    ...
        annotations: [
          { type: 'link' },
          internalLink,
        ],
    ...
  ],
};
```
**To Reproduce**

1. Clone https://github.com/sanity-io/sanity-template-nextjs-landing-pages
2. Run `Run sanity graphql deploy` from the studio folder
3. See error

**Expected behavior**

The GraphQL schema should be deployed.

**Screenshots**

**Which versions of Sanity are you using?**

@sanity/cli             0.142.2 (latest: 0.142.6)
@sanity/base            0.142.2 (latest: 0.142.6)
@sanity/components      0.142.5 (latest: 0.142.6)
@sanity/core            0.142.5 (latest: 0.142.6)
@sanity/default-layout  0.142.5 (latest: 0.142.6)
@sanity/default-login   0.142.3 (latest: 0.142.6)
@sanity/desk-tool       0.142.5 (latest: 0.142.6)

**What operating system are you using?**

macOS Mojave 10.14.6

**Which versions of Node.js / npm are you running?**

npm: 6.10.3
node: v8.12.0

**Comment by saasen** [comment]
Thank you for reporting this! We've gotten around to fix this. It will be deployed in the next release!

PR: https://github.com/sanity-io/sanity/pull/1602

**Comment by rexxars** [comment]
This is now fixed and released as v0.145.0! Thanks again for reporting.
